x-pack/plugins/uptime/server/lib/saved_objects/uptime_settings.ts

```diff
@@ -43,4 +43,20 @@ export const umDynamicSettings: SavedObjectsType = {
         defaultMessage: 'Uptime Settings - Index',
       }),
   },
+  migrations: {
+    // Takes a pre 8.1.0 doc, and converts it to 8.1.0
+    '8.1.0': (doc) => {
+      const { heartbeatIndices } = doc.attributes;
+
+      return {
+        ...doc,
+        attributes: {
+          ...doc.attributes,
+          heartbeatIndices: heartbeatIndices.includes('heartbeat-8*')
```

Comment:
Are we sure these clusters would have `heartbeat-8*` in the pattern? I think they'd only have `heartbeat-7*`. 

I can think of two correct filters:

1. (more conservative) `heartbeatIndices.includes('hearthbeat-7*')`
2. (less conservative) `!heartbeatIndices.includes('synthetics-*')` ?